docs/section5/Rule5-1.md

```diff
@@ -0,0 +1,64 @@
+# Envelope - Rule 5-1  
+**Schema Version** 0.0.33
+**Primary Rule** True
+**Rule ID:** 5-1  
+**Rule Description:** There are four baseline rotations (i.e., four baseline models differing in azimuth by 90 degrees and four sets of baseline model results) if vertical fenestration area per each orientation differ by more than 5%.
+**Rule Assertion:** Options are PASS/FAIL     
+**Appendix G Section:** Table G3.1#5a baseline column   
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None    
+
+**Evaluation Context:** Each RMD 
+
+**Applicability Checks:**  
+None
+   
+**Function Calls:**  
+1. get_opaque_surface_type()  
+
+
+## Rule Logic:   
+
+Get the fenestration area for each unique orientation (i.e., azimuth) and then check if the minimum and maximum areas differ by 5% or more
+- Create a blank dictionary that will have all unique azimuths paired with the total vertical fenestration area: `azimuth_fen_area_dict = {}`  
+- For each building in the B_RMI: `for bldg in B_RMI.buildings:`    
+    - For each building_segment in the bldg: `for bldg_seg in bldg.building_segments`      
+        - For each zone in the building_segment: `for zone in bldg_seg.zones`   
+            - For each surface in zone: `for surface in zone.surfaces:`  
+                - Check if surface is above-grade wall: `if get_opaque_surface_type(surface) == "ABOVE-GRADE WALL":`   
```

Comment:
Do we also need to add surface conditioning category to make sure the surface is one of the conditioned/semi-conditioned type here?

Reply:
@supriyagoel @mkarpman I wondered the same but it does not say anything about that in the 90.1 language so I did not check for that. Surpiya and Maria, do you have an opinion?

Reply:
I agree that based on 90.1 language space conditioning doesn't matter for this determination.

Reply:
@mkarpman thanks for clarifying this logic.